cmd/nanobus/main.go

```diff
@@ -96,6 +97,8 @@ type runCmd struct {
 	BusFile string `name:"bus" default:"bus.yaml" help:"The application configuration or OCI image reference"`
 	// ResourcesFile is the resources configuration (e.g. databases, message brokers).
 	ResourcesFile string `name:"resources" default:"resources.yaml" help:"The resources configuration"`
+	// PackageFile is the resources configuration (e.g. databases, message brokers).
```

Comment:
PackageFile is the OCI Reference.  I didn't think it was appropriate to say the 'BusFile' is accurate because the package also contains the axdl and resources and possibly other static files.